docs/Extensibility_UG/Intro.md

```diff
@@ -0,0 +1,81 @@
+# OpenVINO Extensibility Mechanism {#openvino_docs_Extensibility_UG_Intro}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   openvino_docs_Extensibility_UG_add_openvino_ops
+
+@endsphinxdirective
+
+The Intel® Distribution of OpenVINO™ toolkit supports neural network models trained with multiple frameworks including
+TensorFlow*, Caffe*, MXNet*, Kaldi* and ONNX* file format. The list of supported operations is different for
+each of the supported frameworks. To see the operations supported by your framework, refer to
+[Supported Framework Operations](../MO_DG/prepare_model/Supported_Frameworks_Layers.md).
+
+Custom operations, that is those not included in the list, are not recognized by Model Optimizer out-of-the-box. Therefore, creating Intermediate Representation (IR) for a model using them requires additional steps. This guide illustrates the workflow for running inference on topologies featuring custom operations, allowing you to plug in your own implementation for existing or completely new operations.
+
+If your model contains operations not normally supported by OpenVINO™, the OpenVINO™ Extensibility API lets you add support for those custom operations in a library containing custom operation and use one implementation for Model Optimizer and OpenVINO™ Runtime.
+
+There are two steps to support inference of a model with custom operation(s):
+1. Add support for a [custom operation in the Model Optimizer](../MO_DG/prepare_model/customize_model_optimizer/Customize_Model_Optimizer.md) so
+the Model Optimizer can generate the IR with the operation.
+2. Create a custom operation in it as described in the [Custom Operation](add_openvino_ops.md).
+
+## OpenVINO™ Extensions
+
+An OpenVINO™ provides extensions for:
+
+ * [Custom OpenVINO™ Operation](add_openvino_ops.md):
+    - Enables the use of `ov::Core::read_model` to read Intermediate Representation (IR) with unsupported operations
+    - Enables the creation of `ov::Model` with unsupported operations
```

Comment:
we cannot create ov::Model with unsupported operations. Once we added the extension - it becomes supported.
Probably you wanted to mention that operation is not from OpenVINO opset.